app/jobs/export_dsi_users_to_big_query_job.rb

```diff
@@ -1,13 +1,10 @@
-require "export_dsi_users_to_big_query"
-require "export_dsi_approvers_to_big_query"
```

Comment:
This code is domain-dependant and has been moved from library to service objects. There is no need to require it anymore.